### PR TITLE
Add newline-after-flow-annotation rule

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -94,5 +94,9 @@ module.exports = {
     'react-native/no-unused-styles': ERROR,
     'react-native/no-inline-styles': WARNING,
     'flowtype/require-valid-file-annotation': [ERROR, 'always'],
+    'flowtype/newline-after-flow-annotation': [
+      2,
```
Comment (mrtnzlml) [inline .eslintrc.js]: `ERROR` to be more obvious, please... :)

**Comment by mrtnzlml** [comment]
@tbergq Now the question is - is this rule (one line) really stupid and we should remove it as @oreqizer suggested?

File: .eslintrc.js (hunk 1 above)
Comment (tbergq) [reply .eslintrc.js]: Sorry, copy/paste `rookie-mistake`

**Comment by tbergq** [comment]
Really, its your OCD. For me it is the same

**Pull request merged**